plugins/woocommerce/client/blocks/assets/js/blocks/classic-shortcode/editor.scss

```
@@ -7,7 +7,8 @@
 	border-left: 5px solid #2181d2;
 	padding-left: em(40px);
 }
-.wp-block-woocommerce-classic-shortcode__placeholder .components-placeholder__fieldset {
+.wp-block-woocommerce-classic-shortcode__placeholder
+	.components-placeholder__fieldset {
 	display: grid;
 	grid-template-columns: 1fr;
 }
```

This [lint error](https://github.com/woocommerce/woocommerce/actions/runs/14337473488/job/40188185464?pr=57095) is likely caused by Prettier’s [printWidth: 80](https://github.com/WordPress/gutenberg/blob/e49849c95d115d744ad314a255c2be5394238b82/packages/prettier-config/lib/index.js#L22), which introduced a line break and additional indent. Stylelint doesn’t expect the indent, so it fails.

Rather than disabling Prettier for SCSS or adjusting Stylelint, I suggest nesting the class as a simple and safe fix:

```suggestion
.wp-block-woocommerce-classic-shortcode__placeholder {
	.components-placeholder__fieldset {
		display: grid;
		grid-template-columns: 1fr;
	}
}
```

---

Thank you @nielslange done in https://github.com/woocommerce/woocommerce/pull/57095/commits/9304c1507f4a5ac868bb505291c8b799e246d082